components-core/src/main/java/com/adyen/checkout/components/base/ActionDelegate.kt

```diff
@@ -25,6 +25,11 @@ interface ActionDelegate<ActionT : Action> : ComponentDelegate {
      */
     fun initialize(coroutineScope: CoroutineScope) = Unit
 
+    /**
+     * Override this method if you need to emit to the [exceptionFlow] from outside of this class.
+     */
+    fun onError(e: CheckoutException) = Unit
```

Comment:
Would it be better to use the `InputData` for this?

Reply:
Let me look into that